packages/core/src/domain/npi-facility.ts

```diff
@@ -0,0 +1,82 @@
+import { validateNPI } from "@metriport/shared/common/validate-npi";
+import { defaultOptionalStringSchema, defaultZipStringSchema } from "@metriport/shared/util";
+import z from "zod";
+import { usStateForAddressSchema } from "@metriport/api-sdk";
+
+export enum FacilityType {
+  initiatorAndResponder = "initiator_and_responder",
+  initiatorOnly = "initiator_only",
+}
+
+export const addressStrictSchema = z.object({
+  addressLine1: z.string().min(1),
+  addressLine2: defaultOptionalStringSchema,
+  city: z.string().min(1),
+  state: usStateForAddressSchema,
+  zip: defaultZipStringSchema,
+  country: z.literal("USA").default("USA"),
+});
```

Comment:
I'd add this to a new address.ts file in core and migrate all the types from the original file.